lib/jellyfish/room.ex

```diff
@@ -157,6 +207,62 @@ defmodule Jellyfish.Room do
     {:reply, reply, state}
   end
 
+  @impl true
+  def handle_info(%Message.EndpointMessage{endpoint_id: to, message: {:media_event, data}}, state) do
+    with {:ok, peer} <- Map.fetch(state.peers, to),
+         socket_pid when is_pid(socket_pid) <- Map.get(peer, :socket_pid) do
+      send(socket_pid, {:media_event, data})
+    else
+      nil ->
+        Logger.warn(
+          "Received Media Event from RTC Engine to peer #{inspect(to)} without established signaling connection, room: #{inspect(state.id)}"
+        )
+
+      :error ->
+        Logger.warn(
+          "Received Media Event from RTC Engine to non existent peer (target id: #{inspect(to)}), room: #{inspect(state.id)}"
+        )
+    end
+
+    {:noreply, state}
+  end
+
+  @impl true
+  def handle_info(%Message.EndpointCrashed{endpoint_id: endpoint_id}, state) do
+    Logger.error(
+      "RTC Engine endpoint associated with peer #{inspect(endpoint_id)} crashed, room: #{inspect(state.id)}"
```

Comment:
This would be misleading. Endpoint doesn't always represent a peer (in RTC Engine)